README.md

```diff
@@ -1 +1,74 @@
-# shiny.i18n
\ No newline at end of file
+Repo documentation
+================
+
+<link href="http://fonts.googleapis.com/css?family=Lato:300,700,300italic|Inconsolata" rel="stylesheet" type="text/css"> <link href='docs/style.css' rel='stylesheet' type='text/css'>
+
+shiny.i18n
+==========
+
+Shiny applications internationalization made easy
+
+Change languages and formats easy with shiny.i18n.
+
+<!-- #Basic tutorial article is available on [Appsilon Data Science blog](your_future_art_link). -->
+<!-- Live demo link below 
+<p style="text-align: center; font-size: x-large;">
+<a href="http://appsilondatascience.com/demos">Live demo</a>
+</p>
+-->
+
+Source code
+-----------
+
+This library source code can be found on [Appsilon Data Science's](http://appsilondatascience.com) Github: <br> <https://github.com/Appsilon/shiny.i18n/>
+
+<script>
+document.write('<div class="logo"><a href="http://appsilondatascience.com"><img alt="Appsilon Data Science" src="https://cdn.rawgit.com/Appsilon/website-cdn/gh-pages/logo-white.png"/></a></div>')
```

Comment:
White logo looks nice on documentation website, but on github white page looks like sth is missing.

Reply:
Yeah, I know - probably we need to change that depending on the target. Let me try to fix that.